src/display/editor/annotation_editor_layer.js

```diff
@@ -73,6 +73,13 @@ class AnnotationEditorLayer {
 
   static _initialized = false;
 
+  static #editorTypes = Object.fromEntries(
```

Comment:
Can we please make this a `Map` instead, to avoid having to sprinkle `Object.values(...)` calls in a bunch of places throughout the code?